pkg/storage/v2/mysql/client.go

```diff
@@ -128,6 +137,7 @@ func NewClient(
 		logger.Error("Failed to open db", zap.Error(err))
 		return nil, err
 	}
+	db.SetConnMaxLifetime(dopts.connMaxLifetime)
```

Comment:
I'm adding the setting because it is required to avoid issues where the SQL server could close the connection before the driver.

https://github.com/go-sql-driver/mysql?tab=readme-ov-file#important-settings